### PR TITLE
feat: Time left autobattle

### PR DESCRIPTION
**Pull request by aLIEzsss4**

### CODE REVIEW

File: packages/contracts/src/library/Utils.sol [hunk 1]
```diff
@@ -71,6 +71,40 @@ library Utils {
...
+    uint256 length = Player.lengthHeroes(_player);
+    bytes32 heroId;
+    if ( _index < length) {
+
+      for (uint i = _index; i <(length - 1); i++) {
+        heroId = Player.getItemHeroes(_player, i+1);
+        Player.updateHeroes(_player, i, heroId);
+      }
+
+      heroId = Player.getItemHeroes(_player, length - 1);
```
Comment (ClaudeZsb) [inline packages/contracts/src/library/Utils.sol:96]: after moving behind heroes one slot forward, you always get the last hero but not the expected one at given index.

Comment (aLIEzsss4) [reply packages/contracts/src/library/Utils.sol]: fix [0d446a6](https://github.com/HelheimLabs/autochessia/pull/68/commits/0d446a67f5b2867dae6242628ac093a3f3c8aa5c)

**Review by noyyyy — COMMENTED** [review]
puzzled at game config key. need more research for me.

**Review by noyyyy — APPROVED** [review]
Can we merge it now? @aLIEzsss4

**Pull request merged**